internal/cmd/request.go

```diff
@@ -115,3 +123,75 @@ func PrintCMDResponseFull(request any, response core.APIResponse, responseType a
 		fmt.Println(util.MustToJSONIndent(responseContent))
 	}
 }
+
+// Attempt to convert an API response to a TSV table.
+// Return ("", false) if there are issues converting the API response
+// or (customTable.String(), true) if the response got sucessfully converted.
+func AttemptApiResponseToTable(response core.APIResponse) (string, bool) {
```

Comment:
Name still needs work.
Every function that returns an error (or success status) "attempts" to do something,
so we don't need the prefix here.